packages/components/src/Image/Image.tsx

```diff
@@ -33,5 +33,12 @@ export const Image = ({
     objectFit: fit,
     objectPosition: position,
   };
-  return <Box {...props} as="img" css={css} />;
+  return (
+    <Box
+      {...props}
+      as="img"
+      __baseCSS={fit ? { width: ' 100%', height: '100%' } : {}}
```

Comment:
If we use the fit and position props. We must set the width and height to 100% that the fit and position props can work. 

If we don't had a fit and position props we don't need the width and height  to set to 100% because the image becomes distorted

## Here is a simple Test

```tsx
  <MarigoldProvider theme={theme}>
    <Aspect ratio="widescreen">
      <Image
        fit="contain"
        position="center"
        src="https://images.unsplash.com/photo-1603910234616-3b5f4a6be2b4?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
        alt="event_image"
      />
    </Aspect>
  </MarigoldProvider>
```

### Without width and height: 
<img width="761" alt="image" src="https://user-images.githubusercontent.com/27921741/191697013-917948f0-bd4c-4def-971d-bbcd0cead98b.png">


### With width and height:
<img width="1510" alt="image" src="https://user-images.githubusercontent.com/27921741/191696385-a8a38d9f-0cc2-4379-8485-6d1f3693130c.png">